Automattic/wp-calypso
# Wrap up the plans-domains-swap A/B test

#### Changes proposed in this Pull Request

* Wrap up the plans-domains-swap test. 
* Three iterations of the test was deployed in the PRs - https://github.com/Automattic/wp-calypso/pull/40879, https://github.com/Automattic/wp-calypso/pull/41982, and https://github.com/Automattic/wp-calypso/pull/43675. We are removing the variant entirely and retaining control.

#### Testing instructions



* Go through signup at /start and ensute that everything works well. Also just to be sure, try a few flows such as domain-only at /start/domain, then /start/premium. Also try adding a new site and the site launch flow, and verify that everything works well.

Fixes 629-gh-martech
